Frontend/Frontend.Client/Models/UserMessage.cs

```diff
@@ -11,22 +11,25 @@ public class UserMessage: IMessage
         public long TimeStamp { get; set; }
         public string Hash { get; set; }
         public string Content { get; set; }
+        public long Timestamp { get; set; }
+        public bool IsConfirmed { get; set; }
```

Comment:
There's two timestamp variables now look above

Reply:
removed the additional variable and fixed all references to it to refer to the other one.